src/views/FAQ/FAQ.tsx

```diff
@@ -267,10 +269,11 @@ export const FAQPage: FunctionComponent = () => (
           <FAQItem question="Why isn’t my package displayed on Construct Hub?">
             <Text>
```

Comment:
"Is the content served in Construct Hub meant to be consumed programmatically?" should be updated with the latest content